parsec/core/types/manifest.py

```diff
@@ -54,9 +67,8 @@
 ChunkIDField: Type[fields.Field] = fields.uuid_based_field_factory(ChunkID)
 
 
-@functools.total_ordering
 @attr.s(slots=True, frozen=True, auto_attribs=True, kw_only=True, eq=False)
-class Chunk(BaseData):
+class _PyChunk(BaseData):
```

Comment:
keeping the class but removing half of it implementation ? 🤔

Reply:
`_Py*` classes no more